cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +200,95 @@
+# Exchange certificates.
+#
+# ACME is a protocol that allows automatic renewal of certificates.
+# webpkgserver uses the ACME library from github.com/go-acme/lego
+# to handle certificate renewal. Automatic certificate renewal is enabled
+# in webpkgserver via `SXG.ACME.Enable`. Setting this field to true
+# will enable webpkgserver to automatically request certificate renewals
+# whenever it has determined that the current certificate is expired or about to
+# expire.
+#
+# If you set up more than one instance of webpkgserver, the recommended practice
+# is to have only one instance handle automatic certificate renewals and let
+# other instances reload the fresh certificate from the disk. It minimizes the
+# cost (SXG certificates cost money and we don't need each webpackager instance
+# to have a separate SXG certificate) and prevents the webpkgserver instances
+# from bombarding your Certificate Authority with certificate requests. Note
+# this setup requires you to locate the certificate file on a shared filesystem
+# accessible by all webpkgserver instances. Also note that some Certificate
+# Authorities require that purchased certificates be installed on a single
+# (physical) machine.
+#
+# For the full ACME spec, see:
+#   https://tools.ietf.org/html/rfc8555 
+[SXG.ACME]
+  # Enable webpkgserver to attempt to auto renew certificates using
+  # ACME. Required when ACME is enabled.
+  #Enable = false 
+
+  # The ACME discovery URL. It is specified by the Certificate Authority that
+  # doles out your certificates. As of August 2020, only DigiCert supports
+  # automatic renewals of signed exchange certificate:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
+  # Required when ACME is enabled.
+  DiscoveryURL = '<Your Discovery URL>'
+
+  # The email address registered to the Certificate Authority for your signed
+  # exchange certificates.
+  # Required when ACME is enabled.
+  Email = 'user@example.com'
+
+  # For the remaining configuration items, it's important to understand the
+  # different challenges employed as part of the ACME protocol:
+  # https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#identifier-validation-challenges
+  # https://letsencrypt.org/docs/challenge-types/
+  # https://certbot.eff.org/docs/challenges.html
+  #
+  # Note that you do not need to configure all of these challenges. It is
+  # typically sufficient to configure just one of them. If more than one method
+  # is configured, the go-acme/lego library will decide pick one of them for
+  # primary use and use other settings as backup. For wildcard certificates,
+  # nevertheless, DNSProvider is the only supported method of validation, and
+  # others cannot be used. See DNSProvider for more detail.
+
+  # The http server root directory where the ACME http challenge token should
+  # be deposited. Note that you may need to do some configuration work to
```

Comment:
Configuration = setting one or more of the values in the toml.

Reply:
Can you update the documentation here please?

Reply:
Done.

Reply:
Sorry but I can't tell what is updated...

Reply:
" Note that you do not need to set the fields for all of these challenges. It
  # is typically sufficient to have a setting for just one of the challenges." 

I used "set the fields" instead of "configure".

Reply:
Thanks for your answer, but I was asking about "some configuration work" on this line, as in "... be deposited. Note that you may need to do <u>some configuration work</u> to get this setup to work where ..."

Reply:
Changed.